### PR TITLE
Update README to include additional install steps

### PR DESCRIPTION
**Pull request by chris-erickson**
This was helpfully pointed out in https://github.com/pyca/bcrypt/issues/41 by @jbolda.

The only package I needed was ```libffi-dev```, so maybe someone can confirm the full list.  I also fixed your PyPi badge.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,6 +20,19 @@ To install bcrypt, simply:
 
     $ pip install bcrypt
 
+Note that bcrypt should build very easily on Linux provided you have a C compiler, headers for Python (if you’re not using pypy), and headers for the libffi libraries available on your system.
+
+For Debian and Ubuntu, the following command will ensure that the required dependencies are installed:
+
+.. code:: bash
+
+    $ sudo apt-get install build-essential libssl-dev libffi-dev python-dev
```
Comment (reaperhulk) [inline README.rst]: libssl-dev is not a requirement for bcrypt

File: README.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 bcrypt
 ======
 
-.. image:: https://pypip.in/version/bcrypt/badge.svg?style=flat
+.. image:: https://img.shields.io/pypi/v/nine.svg
```
Comment (reaperhulk) [inline README.rst]: This should be https://img.shields.io/pypi/v/bcrypt.svg

File: README.rst [hunk 3]
```diff
@@ -20,6 +20,19 @@ To install bcrypt, simply:
...
+
+.. code:: bash
+
+    $ sudo apt-get install build-essential libssl-dev libffi-dev python-dev
+
+For Fedora and RHEL-derivatives, the following command will ensure that the required dependencies are installed:
+
+.. code:: bash
+
+    $ sudo yum install gcc libffi-devel python-devel openssl-devel
```
Comment (chris-erickson) [inline README.rst]: @reaperhulk I imagine I should remove ```openssl-devel``` also?

Comment (reaperhulk) [inline README.rst]: Yep :) Thanks for your work here.

**Comment by chris-erickson** [comment]
@reaperhulk Take a look, I implemented the requested changes.

**Comment by reaperhulk** [comment]
Looks good, thanks @chris-erickson!

**Pull request merged**